### PR TITLE
control SIP default room for incoming calls

### PR DESCRIPTION
**Pull request by soletan**
This merge request is adding support for controlling default room to use on incoming SIP calls lacking of proper Invite header to prevent this setting from requiring manual patching of internal configuration file. It is regarding #616.

### CODE REVIEW

**Comment by saghul** [comment]
Sorry for the delay, looks like there is a conflict now, any chance you could take a look?

**Comment by soletan** [comment]
That's obviously due to my other pull request ...

File: env.example [hunk 1]
```diff
@@ -129,6 +129,9 @@ ETHERPAD_SKIN_VARIANTS="super-light-toolbar super-light-editor light-background
 # SIP server transport
 #JIGASI_SIP_TRANSPORT=UDP
 
+# SIP default room name for incoming calls lacking of Invite header
+#JIGASI_SIP_DEFAULT_ROOM=conference
```
Comment (saghul) [inline env.example:133]: Please drop this part, I want to move all the examples to the handbook instead. Otherwise LGTM.

Comment (soletan) [reply env.example]: I see. It's done.

**Review by saghul — APPROVED** [review]

**Pull request merged**